src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -74,18 +74,15 @@ static DatadogLogging()
                 }
 
                 // Ends in a dash because of the date postfix
-                var managedLogPath = Path.Combine(logDirectory, $"dotnet-tracer-{currentProcess.ProcessName}-.log");
+                var managedLogPath = Path.Combine(logDirectory, $"dotnet-tracer.log");
```

Comment:
I didn't know we had this, but now that it's there: Why are we removing it?

Reply:
It was an attempt for me to pre-seed a file to see if I could get managed logging working.
I'll re-add it and figure it out in a later change.